client/ayon_photoshop/plugins/create/create_flatten_image.py

```diff
@@ -145,14 +146,26 @@ def get_product_name(
 
         dynamic_data = prepare_template_data({"layer": "{layer}"})
 
+        get_product_name_kwargs = {}
```

Comment:
Do we need same paragraph of code 3x in different creators instead of function? Or it is just temporary and it will go away shortly? (in that case I am fine with it).